src/scss/components/file/mixins/hxFileControl.scss

```diff
@@ -0,0 +1,66 @@
+@import "vars";
```

Comment:
Import `config` instead. It already imports "vars".

```scss
@import '../config';
```

Reply:
Changes have been done, please review.